client/src/components/navbar.tsx

```diff
@@ -0,0 +1,45 @@
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+
+import { Button } from "@/components/ui/button";
+
+import logo from "../../public/wajo_white.svg";
+import styles from "../styles/modules/navbar.module.css";
+import MobileNav from "./ui/mobilenav";
+
+const Navbar = () => {
+  return (
+    <nav className={`${styles.nav}`}>
```

Comment:
on smaller screen sizes, at around <400px wide, the navbar contents aren't vertically centered and instead stick to the top. Not a huge issue since the main purpose of this app is the competition quizzes and the competition quizzes will always be on desktop but viewing other parts of the website should be responsive. You can fix this by just adding `display: flex` to the nav 

![image](https://github.com/user-attachments/assets/126273b4-e94b-413c-90d0-7cdccfe14dae)